devicetypes/smartthings/Orvibo-Contact-Sensor.src/Orvibo-Contact-Sensor.groovy

```diff
@@ -0,0 +1,150 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *
+ *  Orvibo Contact Sensor
+ *
+ *  Author: Deng Biaoyi
+ *
+ *  Date:2018-07-03
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+
+metadata {
+	definition(name: "Orvibo Contact Sensor", namespace: "smartthings", author: "biaoyi.deng@samsung.com", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
```

Comment:
Try adding mnm n and vid parameters to metadata to avoid creating metadata for that file:
mnmn: "SmartThings",
vid:"generic-contact-3"